docs/postprocessors.rst

```diff
@@ -140,6 +145,57 @@ we can see that the best model for the amount of allocated memory depending on c
 
 .. image:: /../examples/clusterizer-memory-scatter-with-models.*
 
+.. _postprocessors-regressogram:
+
+Regressogram method
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. automodule:: perun.postprocess.regressogram
+
+.. _postprocessors-regressogram-cli:
+
+Command Line Interface
+""""""""""""""""""""""
+
+.. click:: perun.postprocess.regressogram.run:regressogram
+   :prog: perun postprocessby regressogram
+
+.. _postprocessors-regressogram-examples:
+
+Examples
+""""""""
+
+    .. code-block:: json
+
+        {
+            "bin_stats": [
+                13.0,
+                25.5
+            ],
+            "uid": "linear::test2",
+            "bin_method": "doane",
+            "method": "regressogram",
+            "r_square": 0.7575757575757576,
+            "x_interval_end": 9.0,
+            "statistic": "mean",
+            "x_interval_start": 0.0
+        }
+
+The example above shows an example of profile post-processed by regressogram method (note
```

Comment:
Fix: What is missing in this example is (1) explain very briefly what is the "doane" method (maybe with link) and (2) what are the bin stats.